utils/parse.go

```diff
@@ -30,13 +35,17 @@ func ParseShell(input string) []string {
 	}
 
 	args := strings.FieldsFunc(input, f)
+	if count%2 != 0 {
+		return nil, fmt.Errorf("Invalid command: command contains an unpaired quotation mark: '%v'", input)
```

Comment:
no need to display the input, this will likely clutter up the error message users get in slack